ritamma/hometask-1/8.rb

```diff
@@ -0,0 +1,9 @@
+(1..1000).each do |x|
+  (1..1000).each do |y|
```

Comment:
you can use x instead 1, it should be faster ->   (x..1000).each do |y|

Reply:
@evrusha Thanks! I changed it